FightSequence.py

```diff
@@ -0,0 +1,52 @@
+from dunwoody_disaster import CharacterFactory, Item
+from dunwoody_disaster.views import fightScreen
+
+
+class FightSequence:
+    def __init__(self, player, enemy):
+        self.player = player
+        self.enemy = enemy
+
+    def Fight(self, playerWeapon, enemyWeapon):
```

Comment:
```suggestion
    def Fight(self, playerWeapon: Item.Weapon, enemyWeapon: Item.Weapon) -> tuple:
```

Reply:
There is some way to make this like `-> tuple[CharacterFactory.Character]` but I don't remember exactly how.